hudi-client/hudi-client-common/src/main/java/org/apache/hudi/io/HoodieAppendHandle.java

```diff
@@ -107,6 +107,8 @@
   // Header metadata for a log block
   private final Map<HeaderMetadataType, String> header = new HashMap<>();
   private SizeEstimator<HoodieRecord> sizeEstimator;
+  // denotes GCSFileSystem
+  private boolean isGCSFileSystem;
```

Comment:
lets not leak this into the handle classes. these should strictly be storage agnostic